config/shakapacker.yml

```diff
@@ -23,7 +23,7 @@ development:
   # Reference: https://webpack.js.org/configuration/dev-server/
   dev_server:
     https: false
-    host: localhost
```

Comment:
`0.0.0.0` is required, as the container is not in the same space as the host.

For truly remote, such as CodeSpaces, one might also need to change the allowed list below